### PR TITLE
add identifier panel variable types

### PR DESCRIPTION
**Pull request by ix0rai**
also fixes superclass field to use the full name

### CODE REVIEW

File: enigma/src/test/java/org/quiltmc/enigma/input/bridge/BaseClass.java [hunk 1]
```diff
@@ -62,6 +62,10 @@ public BaseClass baz(int xz) {
 		return new BaseClass(xz, 0, xz);
 	}
 
+	public BaseClass whee(long xyz, int q, int w) {
```
Comment (IotaBread) [inline enigma/src/test/java/org/quiltmc/enigma/input/bridge/BaseClass.java:65]: I hope this didn't break any tests

**Review by IotaBread — APPROVED** [review]

File: enigma/src/test/java/org/quiltmc/enigma/input/bridge/BaseClass.java (hunk 1 above)
Comment (ix0rai) [reply enigma/src/test/java/org/quiltmc/enigma/input/bridge/BaseClass.java:65]: oh no the stats

Comment (ix0rai) [reply enigma/src/test/java/org/quiltmc/enigma/input/bridge/BaseClass.java:65]: oh god oh no

Comment (IotaBread) [reply enigma/src/test/java/org/quiltmc/enigma/input/bridge/BaseClass.java:65]: It didn't

Comment (ix0rai) [reply enigma/src/test/java/org/quiltmc/enigma/input/bridge/BaseClass.java:65]: yay :D

**Pull request merged**